File: rna-transcription/rna_transcription.py
def to_rna(dna):
    rnadict = {
        "G": "C",
        "C": "G",
        "T": "A",
        "A": "U"
        }

    try:
        transcribed = "".join([rnadict[x] for x in dna])

    except KeyError:
        transcribed = ""

    return transcribed

File: challenges/exercism/javascript/easy/rna-transcription.js
'use strict';

/** Translates a DNA sequence to RNA. */
export const toRna = (sequence) => {
  const pairs = { G: 'C', C: 'G', T: 'A', A: 'U' };
  return [...sequence].map((x) => pairs[x]).join('');
};
